e2e/nx-plugin-e2e/mocks/utils.ts

```diff
@@ -0,0 +1,27 @@
+import {join, relative} from "node:path";
+import {executeProcess} from "@code-pushup/utils";
+
+export function distPluginPackage(cwd: string): string {
+  return relative(join(process.cwd(), cwd), join(process.cwd(), "dist/packages/nx-plugin"));
+}
+export function pluginFilePath(cwd: string): string {
+  return relative(join(process.cwd(), cwd), join(process.cwd(), "packages/nx-plugin/src/index.ts"));
+}
```

Comment:
To e2e test the build output we have to use a relative path to the dist folder. 
If we use `@code-pushup/nx-plugin` as `bin`, verdaccio throws: `// npm ERR! 404 '@code-pushup/nx-plugin@0.29.0' is not in this registry`. 
We can also not use the published package as our [e2e tests need a refactoring](https://github.com/code-pushup/cli/issues/643)